virt/virtual_machines/virt-managing-configmaps-secrets-service-accounts.adoc

```diff
@@ -0,0 +1,26 @@
+[id="virt-managing-configmaps-secrets-service-accounts"]
+= Managing ConfigMaps, secrets, and service accounts in virtual machines
+include::modules/virt-document-attributes.adoc[]
+:context: virt-managing-configmaps-secrets-service-accounts
+toc::[]
+
+
+Secrets, ConfigMaps, and service accounts are Kubernetes objects that are integrated into {product-title}. You can use them to pass configuration data to virtual machines. For example, to enable a virtual machine to access a service that requires credentials, add a secret to the virtual machine.
```

Comment:
"Secrets, ConfigMaps, and service accounts are Kubernetes objects that are integrated into {product-title}"
I believe the message is that OpenShift is Kubernetes, so this sentence seems redundant. 

I don't often get to say this but I think I want more information :)
I would find it helpful to provide a (brief) use case for each of the objects, much like the example given for secrets. . For instance, I have a ~reasonable idea of what a configmap is but I had to go looking to see how I might find it useful to add it to a VM.